Problem uploading videos on Nextcloud for Android

### Steps to reproduce
1. Open settings
2. Auto upload
3. Configure for video uploads
4. Set path and hit save

### Expected behaviour
- should simply start uploading videos from my device to  the chosen path in my next cloud

### Actual behaviour
- ************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.preview.PreviewImageActivity}: java.lang.ClassCastException: android.accounts.Account cannot be cast to com.nextcloud.client.account.User
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4031)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4197)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2434)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8633)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1133)
Caused by: java.lang.ClassCastException: android.accounts.Account cannot be cast to com.nextcloud.client.account.User
	at com.owncloud.android.ui.activity.FileActivity.onCreate(FileActivity.java:209)
	at com.owncloud.android.ui.preview.PreviewImageActivity.onCreate(PreviewImageActivity.java:107)
	at android.app.Activity.performCreate(Activity.java:8282)
	at android.app.Activity.performCreate(Activity.java:8262)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4005)
	... 12 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30190090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: r7
Model: SM-N770F


************ FIRMWARE ************
SDK: 31
Release: 12
Incremental: N770FZHU8FVB4


### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

### Logs
#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

--- a/src/main/java/com/owncloud/android/files/services/FileDownloader.java
+++ b/src/main/java/com/owncloud/android/files/services/FileDownloader.java
@@ -574,7 +574,7 @@ public class FileDownloader extends Service
             showDetailsIntent = new Intent(this, FileDisplayActivity.class);
         }
         showDetailsIntent.putExtra(FileActivity.EXTRA_FILE, download.getFile());
-        showDetailsIntent.putExtra(FileActivity.EXTRA_USER, download.getAccount());
+        showDetailsIntent.putExtra(FileActivity.EXTRA_USER, download.getUser());
         showDetailsIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
 
         mNotificationBuilder.setContentIntent(PendingIntent.getActivity(this, (int) System.currentTimeMillis(),

